src/org/usfirst/frc4904/robot/subsystems/MilkIO.java

```diff
@@ -0,0 +1,26 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class MilkIO extends Subsystem {
```

Comment:
Realistically we should probably rename this to CubeIO or BoxIO. @ajnadel any input? I'm going to tentatively approve this in the meantime.

Reply:
As cute as `MilkIO` is, I think it should be one of the others.